impl/src/main/java/io/kroxylicious/testing/kafka/common/KafkaClusterConfig.java

```diff
@@ -571,6 +576,11 @@ public String clusterId() {
         return isKraftMode() ? kafkaKraftClusterId : null;
     }
 
+    private static String detectKafkaVersionFromClasspath() {
+        var version = AppInfoParser.getVersion();
+        return version == null || version.equals("unknown" /* AppInfoParser.DEFAULT_VALUE */) ? Version.LATEST_RELEASE : version;
```

Comment:
Did you mean to leave the reference to AppInfoParser?

Reply:
Are you referring to the comment `/* AppInfoParser.DEFAULT_VALUE */`?  My preference would have been to use their constant, AppInfoParser.DEFAULT_VALUE, but that's package private.  I stuck in the comment with the intent of making it clean where "unknown" comes from.